src/toICN-core.js

```diff
@@ -60,28 +72,26 @@ exports.toICN = function(raw){
   }
   return s;
 };
+
 exports.updateChords = function(keyChords){
   keyChords.forEach((e) => {
     if(e.type == "key"){
       // 転調の場合
       if(isAutoKeyDetection){
-        keyMatch = e.v.match(/(: |：)([A-G](#|b){0,1})(m{0,1})$/);
-        key = keyMatch?sharpify(keyMatch[2]):"";
-        keyMinorSignature = keyMatch?keyMatch[4]:"";
-        let keyNo = scale.indexOf(sharpify(key));
-        if(keyMinorSignature=="m"){keyNo += 3;}
-        if(previousKeyNo != -1){
-          let keyModulationDegree = keyNo - previousKeyNo;
+        keyMatch = e.v.match(/(: |：)([A-G](#|b){0,1}m{0,1})$/);
```

Comment:
元もついてなかったですが、ローカル変数にしたいです。

```suggestion
        let keyMatch = e.v.match(/(: |：)([A-G](#|b){0,1}m{0,1})$/);
```

Reply:
keyMatch だとグローバル変数のkeyMatchと名前が衝突するので、名前を変えた上でローカル変数にしました。